Minesweeper.py

```diff
@@ -1,40 +1,31 @@
 import pygame
 import random
+pygame.init()
 
-# DHB: it is somewhat unnatural to see pygame used before being initialised. Also, it is not cool to execute code in multiple parts
-# DHB: of the file. Define your classes and functions first, then execute your code.
-# Create the game window
-win = pygame.display.set_mode((888, 938))
-pygame.display.set_caption("Minesweeper")
-
-# DHB: all classes in python should contain class comments, rather than preceeding the class with a comment like this
-# Define the Color class to store color values
 class Color(object):
-    # DHB: it is not great to skip the parenthesis around a tuple, even though it is legal. If you would e.g. copy this constant
-    # DHB: elsewhere, such as in a function parameter list, then it would change from being a tuple to being multiple parameters 
-    GRAY = 155, 155, 155
-    BACKGROUND = 120, 120, 120
-    BLACK = 0, 0, 0
-    WHITE = 255, 255, 255
-    DARK_GRAY = 100, 100, 100
-    RED = 255, 0, 0
-    BLUE = 0, 0, 255
-    GREEN = 0, 255, 0
-    DARK_BLUE = 0, 0, 139
-    MAROON = 128, 0, 0
-    TURQUOISE = 48, 213, 200
-
-# DHB: incorrect approach to commenting a class
-class TutorialScreen(object): # creates the tutorial screen
+    """ Define the Color class to store color values
```

Comment:
The comment of a class should generally read as if the class is an object being explained to somebody else on your team. It would be best that the comment explains what, why, and how.

e.g. "The Color class is used to store our color constants, so that we do not use hardcoded color values all over the place, which is a bad practice"

what: the object under discussion is "The color class".
why: so that we do not use hardcoded color values all over the place, which is a bad practice
how: we store them in constants